docs-rtd/index.md

```diff
@@ -26,13 +26,18 @@ Like all of Juju, the Terraform Provider for Juju is for SREs, or anyone looking
 
 ## In this documentation
 
-````{grid} 1 1 2 2
+- **Set up Juju:** {ref}`manage-the-terraform-provider-for-juju`, {ref}`manage-controllers`, {ref}`manage-models`
```

Comment:
i would perhaps split this into the following bullet points:
- set up the juju terraform provider: `manage-the-terraform-provider-for-juju`
- connect a cloud: `manage-clouds`, `manage-credentials`
- deploy infrastructue: `deploy-an-application`, `configure-an-applicaton`, `integrate-an-applicaton-with-another-application`, `scale-an-application`, `manage-machines`, `manage-secrets`
- authorization: `manage-ssh-keys`, `manage-users`, `manage-service-accounts`, `manage-groups`, `manage-roles`, `maange-controllers`, `manage-models`